docs/content/en/install/local/_index.md

```diff
@@ -1,441 +1,2316 @@
+
+```bash
+kubectx kind-k8ssandra-1
+
+$ kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -- /bin/bash
+```
+
+**Output plus nodetool example:**
+
+```bash
+Defaulted container "cassandra" out of: cassandra, server-system-logger, jmx-credentials (init), server-config-init (init)
+cassandra@k8ssandra-1-worker2:/$ nodetool -u demo-superuser -pw KT-ROFfbD-O9BzWS3Lxq status
+Datacenter: dc1
+===============
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address      Load        Tokens  Owns (effective)  Host ID                               Rack
+UN  172.18.0.8   106.28 KiB  16      100.0%            676592e3-72f2-40c7-893e-932ab7f99fdd  default
+UN  172.18.0.9   101.93 KiB  16      100.0%            a6b62d88-0223-432d-a040-e10ce93a3342  default
+UN  172.18.0.7   106.26 KiB  16      100.0%            1228c0ec-b5f3-4bbc-9b32-e32a8600e58b  default
+
+Datacenter: dc2
+===============
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address      Load        Tokens  Owns (effective)  Host ID                               Rack
+UN  172.18.0.14  106.2 KiB   16      100.0%            0836237c-66d7-453a-b03a-91d64aaa0f77  default
+UN  172.18.0.12  106.18 KiB  16      100.0%            592a453f-7abb-483c-ab4e-2f37c8cd4713  default
+UN  172.18.0.16  106.2 KiB   16      100.0%            d19b5028-b28e-450b-a417-1cd2e42aa179  default
+
+Datacenter: dc3
+===============
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address      Load        Tokens  Owns (effective)  Host ID                               Rack
+UN  172.18.0.21  101.74 KiB  16      100.0%            2cceff49-6df2-4045-8e04-6ce262bd6fc4  default
+UN  172.18.0.20  106.11 KiB  16      100.0%            38438f65-b10a-4b3f-a56f-488536bf4cd3  default
+UN  172.18.0.19  106.1 KiB   16      100.0%            018bfd17-8a77-43c6-859b-ab69c1fc8a66  default
+```
+
+{{% alert title="Tip" color="success" %}}
+All nodes should have the status UN, which stands for "Up Normal".
+{{% /alert %}}
+
+### Test a few operations
+
+```bash
+kubectx kind-k8ssandra-1
+
+kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -- /bin/bash
+```
+
+**Output and cqlsh example:**
+
+In the launched container's `cqlsh` session, notice we provide the extracted password for `demo-superuser`.
+
+```bash
+Defaulted container "cassandra" out of: cassandra, server-system-logger, server-config-init (init)
+cassandra@k8ssandra-1-worker2:/$ cqlsh -u demo-superuser -p KT-ROFfbD-O9BzWS3Lxq
+Connected to demo at 127.0.0.1:9042
+[cqlsh 6.0.0 | Cassandra 4.0.1 | CQL spec 3.4.5 | Native protocol v5]
+Use HELP for help.
+demo-superuser@cqlsh> describe keyspaces;
+
+data_endpoint_auth  system_auth         system_schema  system_views
+system              system_distributed  system_traces  system_virtual_schema
+
+demo-superuser@cqlsh> CREATE KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'dc1' : 3, 'dc2' : 3, 'dc3': 3};
```

Comment:
The comment above related to breaking these commands out into non-interactive shell commands applies here as well.

If you do indeed want to swap them out, just be sure to get the right replication strategy since they are different in the examples between the single and multi cluster examples.